identity-account/examples/methods.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example methods
+
+use identity_account::account::Account;
+use identity_account::error::Result;
+use identity_account::events::Command;
+use identity_account::identity::IdentitySnapshot;
+use identity_account::storage::MemStore;
+use identity_did::verification::MethodScope;
+use identity_iota::did::IotaDID;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+  pretty_env_logger::init();
+
+  // Create an in-memory storage instance for the account
+  let storage: MemStore = MemStore::new();
+
+  // Create a new Account with the default configuration
+  let account: Account<MemStore> = Account::new(storage).await?;
+
+  // Create a new Identity with default settings
+  let snapshot: IdentitySnapshot = account.create(Default::default()).await?;
+
+  // Retrieve the DID from the newly created Identity state.
+  let document: &IotaDID = snapshot.identity().try_document()?;
+
+  // Add a new Ed25519 (defualt) verification method to the identity - the
+  // verification method is included as an embedded authentication method.
+  let command: Command = Command::create_method()
```

Comment:
Would it perhaps be possible to keep track of the command inside the account as well? Then we can have an array of unpublished commands and when you call `account.update(DID).await?;` it publishes all the commands in a single message. This is more efficient on the Tangle as you can easily group commands and easier for the developers as they don't have to keep track of commands.

Reply:
follow-up PR 😛